src/docs/development/tools/hot-reload.md

```diff
@@ -360,7 +360,7 @@ widgets and render objects.
 
 
 [const-new]: https://news.dartlang.org/2012/06/const-static-final-oh-my.html
-[Dart Virtual Machine (VM)]: https://dart.dev/platforms
+[Dart Virtual Machine (VM)]: {{site.dart-site}}/platforms
```

Comment:
We've removed this page, fyi... It'll get redirected to https://dart.dev/overview#platform.